types/fundamental/real32quant/README.md

```diff
@@ -0,0 +1,20 @@
+# Real Column Types with variable width
+
+## Fields
+
+ * `FloatReal32Quant2_-1_1`
+ * `DoubleReal32Quant2_-1_1`
+ * `DoubleReal32Quant20_-1_1`
+ * `FloatReal32Quant8_0_1`
+ * `FloatReal32Quant16_0_1`
```

Comment:
Can these also use `[-1, 1]` so that we can omit the value range for most of the fields?